### PR TITLE
#273 Feature/ellipsis menu on the booking page

### PR DESCRIPTION
**Pull request by olena-kozak**
https://user-images.githubusercontent.com/98499085/189355578-e2b1970a-9e16-461c-89f7-743cc15a6ce2.MP4

We don't have edit meeting page, so handling edit button is absent from this pull request

### CODE REVIEW

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs [hunk 1]
```diff
@@ -79,7 +79,7 @@ private static List<UserMeetingDTO> GetAllParticipants(Meeting meeting, int numb
                 });
 
             return slotMembers.Union(external).Take(numberOfMembers).ToList();
-        } 
+        }
```
Comment (PublicFedorenko) [inline backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs:82]: Missing new line

**Review by PublicFedorenko — CHANGES_REQUESTED** [review]

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs [hunk 2]
```diff
@@ -79,7 +79,7 @@ private static List<UserMeetingDTO> GetAllParticipants(Meeting meeting, int numb
...
         private static string CreateMemberTitle(Meeting meeting)
```
Comment (PublicFedorenko) [inline backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs]: Missing new line

**Review by PublicFedorenko — CHANGES_REQUESTED** [review]

**Review by PublicFedorenko — APPROVED** [review]

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs (hunk 2 above)
Comment (apofis96) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs]: new line is still missing

File: frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts [hunk 3]
```diff
@@ -94,17 +149,21 @@ export class BookingsPageComponent extends BaseComponent implements OnInit {
...
+        }
     }
 
     private getPageSize(): number {
         const bookingContainer = this.el.nativeElement.getElementsByClassName('container')[0] as HTMLElement;
 
         return bookingContainer.offsetWidth;
     }
+
+    private reloadBookings() {
```
Comment (apofis96) [inline frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts]: Why do we need this extra method with one call to another method&

**Review by apofis96 — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts (hunk 3 above)
Comment (olena-kozak) [reply frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts]: Fixed it.

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs (hunk 2 above)
Comment (olena-kozak) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs:84]: I think smth is wrong is with git displaying changes. In branch it looks good.
image

Comment (olena-kozak) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs]: Fixed it.

File: frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts [hunk 4]
```diff
@@ -83,6 +126,18 @@ export class BookingsPageComponent extends BaseComponent implements OnInit {
         window.open(link);
     }
 
+    private loadMeetings(meetingMemberRequest: IMeetingMembersRequest) {
+        this.meetingService
+            .getThreeMeetings(meetingMemberRequest)
```
Comment (tatianahutii) [inline frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts]: We need to change method name

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by apofis96 — APPROVED** [review]

File: frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts (hunk 4 above)
Comment (olena-kozak) [reply frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts]: Changed method name to getMeetings(meetingMemberRequest).

File: frontend/package.json [hunk 5]
```diff
@@ -27,6 +27,7 @@
         "@angular/router": "^14.1.1",
         "@microsoft/signalr": "^6.0.7",
         "@ng-select/ng-select": "^9.0.2",
+        "@ngtools/webpack": "^14.2.2",
```
Comment (tatianahutii) [inline frontend/package.json]: Why do we need this package?

File: frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts [hunk 6]
```diff
@@ -36,12 +43,24 @@ export class BookingsPageComponent extends BaseComponent implements OnInit {
...
 
     public ngOnInit(): void {
+        this.getBookings();
+    }
+
+    override ngOnDestroy(): void {
+        super.ngOnDestroy();
+
+        this.deleteEventSubscription.unsubscribe();
+    }
```
Comment (tatianahutii) [inline frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts]: Please move this to the end оf the file

File: frontend/yarn.lock [hunk 7]
```diff
@@ -1953,6 +1953,11 @@
   resolved "https://registry.yarnpkg.com/@ngtools/webpack/-/webpack-14.1.1.tgz#a44846f337dd5077649b0b6e561707925db580a0"
   integrity sha512-pj8sN6jBIi2otTHE/CNQyy09Pmn8tGsZyFrSiNO147yjLnrOSJTeFuXfE2pYgrmcYgj0ybnK1zstt4bAKaL7/Q==
 
+"@ngtools/webpack@^14.2.2":
+  version "14.2.2"
+  resolved "https://registry.yarnpkg.com/@ngtools/webpack/-/webpack-14.2.2.tgz#f001cc7373e88c3540cfb069713ed48f889d1bc0"
+  integrity sha512-mjHv5DPsK6QP/D4zi6Oc4gMEZGNYv6Hmw/xJ1ZZTjSxgKujmHx2keDLSJHZpoLtbHRr+FHfWNnPfPz9dWYrlYQ==
+
```
Comment (tatianahutii) [inline frontend/yarn.lock]: See the previous comments

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/yarn.lock (hunk 7 above)
Comment (olena-kozak) [reply frontend/yarn.lock]: Removed it.

File: frontend/package.json (hunk 5 above)
Comment (olena-kozak) [reply frontend/package.json]: Removed it.

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**